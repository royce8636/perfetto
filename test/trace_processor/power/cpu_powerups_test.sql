SELECT IMPORT("chrome.cpu_powerups");
SELECT * FROM chrome_cpu_power_first_toplevel_slice_after_powerup;
