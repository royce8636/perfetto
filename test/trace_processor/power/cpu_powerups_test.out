
"slice_id","previous_power_state"
424,2
703,2
708,2
